### PR TITLE
Add possibility for disabling "Sequence Filtering" globally

### PR DESCRIPTION
**Pull request by GithubMood**

### CODE REVIEW

File: rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java [hunk 1]
```diff
@@ -175,6 +175,24 @@ public void createDataSet(DataSetConfig dataSetConfig) {
...
+            return false;
+        }
+
+        return dataSetConfig.isUseSequenceFiltering();
+    }
+
+    private boolean isSequenceFilteringDisabled(DBUnitConfig dbUnitConfig) {
+        if (dbUnitConfig.getProperties().containsKey("disableSequenceFiltering")) {
+            Boolean globalUseSequenceFiltering = (Boolean) dbUnitConfig.getProperties().get("disableSequenceFiltering");
+            return Boolean.TRUE.equals(globalUseSequenceFiltering);
```
Comment (rmpestano) [inline rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java:190]: We can just return globalUseSequenceFiltering directly, right?

**Review by rmpestano — CHANGES_REQUESTED** [review]
Thank you for the contribution @GithubMood!

**Comment by rmpestano** [comment]
I've committed some adjustment to your branch, please have a look if it keeps with your original idea and let me know if we can merge

File: rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java (hunk 1 above)
Comment (GithubMood) [reply rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java]: My bad. 

I wanted to write:
```
        if (dbUnitConfig.getProperties().containsKey("disableSequenceFiltering")) {
            String useSequenceFiltering = (String) dbUnitConfig.getProperties().get("disableSequenceFiltering");
            return Boolean.parseBoolean(useSequenceFiltering);
        }
```
to be more polite for incorrect configuration. 

If somebody will make a mistake in the configuration, we will just ignore the value and treat disableSequenceFiltering as false. 


For instance the broken configuration:
```
  disableSequenceFiltering: truee
```

will give the exception:
Caused by: java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Boolean

with direct Boolean retrieving

@rmpestano What do you think?

**Comment by GithubMood** [comment]
Thank you! 

No question from my side. Looks perfect.

File: rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java (hunk 1 above)
Comment (rmpestano) [reply rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java]: I think in that case we will get an error even earlier when we try to map the configuration cause now I've added an explicitly boolean property here: https://github.com/database-rider/database-rider/pull/251/commits/8abcd32f3e7a7876ba790e54a3c966622d97cc34#diff-445de3efd40b9ea8307d494417571f8dd6b4fb3da0a70fdd2cf9d82323c2c40dR34

**Pull request merged**